### DISCUSSION TITLE
Stuck at "79% Applying Thehive Salt State"

### DISCUSSION THREAD

**Discussion by selienk** [Security-Onion-Solutions/securityonion#1855] [Unsupported Versions]
I just down Security Onion 2.3.2 iso and install clean. But it allway stuck at "79% Applying Thehive Salt State"
How i can fix it ?

![image](https://user-images.githubusercontent.com/70885510/98515872-56b7df00-229e-11eb-8ade-d0d1925f4eeb.png)

**Comment by weslambert** [comment]
What kind of hardware are you installing this on, and what kind of resources are available to the hardware?

**Comment by selienk** [comment]
Hi Weslambert,

I run on VM (proxmox), 16GB RAM and 100GB HDD.

Dang The Van (Mr) | SYS-SGN Dept
(T) 090.999.3811 - 28.3911.8200 - Ext: 163
(E) sys-sgn@netnam.vn - www.netnam.vn
-
